srcpkgs/marvin/template

```diff
@@ -0,0 +1,56 @@
+# Template for Marvin
+pkgname=marvin
+version=18.16.0
+revision=1
+maintainer="Brenton Horne <brentonhorne77@gmail.com>"
+homepage="https://chemaxon.com/products/marvin"
+license="marvin"
+depends="oracle-jdk"
+short_desc="Skeletal structure drawing program by ChemAxon"
+only_for_archs="x86_64"
+repository=nonfree
+restricted=yes
+nostrip=yes
+
+_baseUrl="https://dl.chemaxon.com/marvin/"
```

Comment:
This does never get used, remove it.

Reply:
All four lines? Or just the _baseUrl? I've deleted that in my last commit, but not the nostrip or restricted lines.

Reply:
only _baseUrl.